056_digitToWords.js

```diff
@@ -0,0 +1,27 @@
+const convertDigitToWords = (number) => {
+  let numberInWords = "";
+  let words = [
+    "zero",
+    "one",
+    "two",
+    "three",
+    "four",
+    "five",
+    "six",
+    "seven",
+    "eight",
+    "nine",
+  ];
+  while (number > 0) {
+    let digit = number % 10;
+    for (let i = 0; i < words.length; i++) {
+      if (i === digit) {
+        numberInWords = words[i] + " " + numberInWords;
+      }
+    }
+    number = Math.floor(number / 10);
+  }
+  return numberInWords;
+};

```

Comment:
add a space here. After main function, keep adding a space would be good for readability.